addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/DefaultExplorePanel.java

```diff
@@ -0,0 +1,221 @@
+        panel.add(new JLabel(" "), LayoutHelper.getGBC(0, 2, 2, 1.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5))); // Spacer
+        return panel;
+    }
+
+    @Override
+    public JPanel getContentPanel() {
+        if (contentPanel == null) {
+            contentPanel = new JPanel(new GridBagLayout());
+            contentPanel.setBackground(Color.WHITE);
+            int formPanelY = 0;
+
+            JPanel savePanel = QuickStartHelper.getHorizontalPanel();
+
+            savePanel.add(new JLabel(Constant.messages.getString("quickstart.explore.1.start")));
+            savePanel.add(getSaveButton());
+            savePanel.add(new JLabel(Constant.messages.getString("quickstart.explore.1.end")));
+            contentPanel.add(savePanel,
+                    LayoutHelper.getGBC(0, formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+
+            contentPanel.add(new JLabel(Constant.messages.getString("quickstart.explore.2")),
+                    LayoutHelper.getGBC(0, ++formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+
+            JPanel proxyPanel = QuickStartHelper.getHorizontalPanel();
+            proxyPanel.add(new JLabel(Constant.messages.getString("quickstart.explore.3")));
+            proxyPanel.add(getHostPortField());
+            proxyPanel.add(getCopyButton());
+            contentPanel.add(proxyPanel,
+                    LayoutHelper.getGBC(0, ++formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+            contentPanel.add(new JLabel(""),
+                    LayoutHelper.getGBC(0, ++formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+        }
+
+        return contentPanel;
+    }
+
+    public JButton getSaveButton() {
+        if (saveButton == null) {
+            saveButton = new JButton(Constant.messages.getString("menu.file.save"));
+            saveButton.setIcon(
+                    DisplayUtils.getScaledIcon(new ImageIcon(ZAP.class.getResource("/resource/icon/16/096.png"))));
+            saveButton.addActionListener(new ActionListener() {
+
+                @Override
+                public void actionPerformed(ActionEvent arg0) {
+                    final JFileChooser fc = new JFileChooser(System.getProperty("user.home"));
+                    fc.setFileSelectionMode(JFileChooser.FILES_ONLY);
+                    fc.setMultiSelectionEnabled(false);
+                    fc.setSelectedFile(new File(OWASP_ZAP_ROOT_CA_FILENAME));
+                    if (fc.showSaveDialog(DefaultExplorePanel.this) == JFileChooser.APPROVE_OPTION) {
+                        final File f = fc.getSelectedFile();
+                        try {
+                            writePubCertificateToFile(f);
+                        } catch (final Exception e) {
+                            View.getSingleton().showWarningDialog(DefaultExplorePanel.this, Constant.messages
+                                    .getString("quickstart.explore.warning.savefail".concat(e.getMessage())));

```

Comment:
The resource message does not seem to exist, it's also concatenating with the key instead of the resource message.